unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/WebSocketCommunication.cs

```diff
@@ -0,0 +1,235 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.NetworkInformation;
+using DCL;
+using UnityEngine;
+using WebSocketSharp.Server;
+public class WebSocketCommunication : IKernelCommunication
```

Comment:
```suggestion

public class WebSocketCommunication : IKernelCommunication
```
Please add a single /n between class and method definitions.